app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -306,7 +305,6 @@ def activation_key_params
         key_params[:unlimited_content_hosts] = false if max_hosts
         key_params[:max_content_hosts] = nil if unlimited
       end
-
```

Comment:
If they aren't relative or required for the change, I prefer they not be included.